README.md

```diff
@@ -6,6 +6,14 @@
 
 This will provide you with a starting Drupal project that is managed with Composer. The install will include a small set of contrib modules, a starting custom module for specific for the build, and a custom starting theme generated from Emulsify.
 
+# Installation
+
+## Requirements
+Without these you will have difficulty installing this project.
+
+1. [PHP 7.4](http://www.php.net/)
+2. [Node ~12.13.0 \(we recommend NVM\)](https://github.com/creationix/nvm)
+3. [Composer](https://getcomposer.org/)
```

Comment:
@mikeethedude If we are specifying versions we might want to go ahead and specify version 2 here.